app/Http/Controllers/Auth/AuthController.php

```diff
@@ -190,9 +109,8 @@ public function emailChangeStep1(Request $request)
     {
         Validator::make(
             $request->all(),
-            ['email' => 'required|email', 'URIstep1' => 'required', 'URIstep2' => 'required']
-        )->validate()
-        ;
+            ['email' => 'required|email', 'uri_step1' => 'required', 'uri_step2' => 'required']
```

Comment:
![CRITICAL](https://sonarsource.github.io/sonar-github/severity-critical.png 'Severity: CRITICAL') Define a constant instead of duplicating this literal "email" 14 times. [![rule](https://sonarsource.github.io/sonar-github/rule.png)](https://sonarcloud.io/coding_rules#rule_key=php%3AS1192)